nishant_work/private.py

```diff
@@ -0,0 +1,20 @@
+#How to write private class in python? give program for example.
```

Comment:
This entire answer is for how to create private and protected variable. but the question is how to create private class in python not variable